trappy/base.py

```diff
@@ -206,6 +206,16 @@ def generate_data_dict(self, data_str):
         prev_key = None
         for field in data_str.split():
             if "=" not in field:
+                if not prev_key:
+                    if 'FAILED TO PARSE' in data_str:
+                        raise ValueError(
+                            'trace-cmd failed to parse the "{}" event. You may '
+                            'need to compile the latest trace-cmd and put it in '
+                            'your $PATH'.format(self.__class__.__name__))
```

Comment:
I think `self.__class__.unique_word` would be more accurate as it will give the exact event name. Class name is already pretty good though, it'll be roughly the same thing but in CamelCase.